src/app/add-note/add-note.component.html

```diff
@@ -0,0 +1,33 @@
+<h1>Add a note</h1>
+<label>Title</label>
+<form [formGroup]="form">
+  <mat-form-field>
+    <input
+      matInput
+      name="inputNote"
+      placeholder="Enter your title"
+      type="text"
+      formControlName = "title"
+      required
+    />
+  </mat-form-field>
+  <div *ngIf="title2.errors?.['required']">
```

Comment:
Mi-e imi dadea eroare la linia asta, a trebuit sa modific codul cu `*ngIf="title2.errors?.required"` ca sa imi build-uiasca

Reply:
Mie doar asa imi build-uieste. Daca pun ca tine imi zice eroarea asta: Property 'required' comes from an index signature, so it must be accessed with ['required'].